### ISSUE TITLE
USWDS - Bug: Tooltip accessibility tests pass screen reader check

### ISSUE THREAD

**Issue by coforma-jamie** [uswds/uswds#6120]
### Describe the bug

The [Tooltip accessibility tests page](https://designsystem.digital.gov/components/tooltip/accessibility-tests/) says that tooltips fail the "Screen reader announces tooltip content" test. However, this isn't the case - I tried it on my own with VoiceOver, and Alexandra Hull noted that "that is an error on our end :sweat_smile:. There's some enhancements related to this specific criteria, but it's not a true failure," and asked me to report the "failure" as a bug.

### Steps to reproduce the bug

1. Go to the [Tooltip accessibility tests page](https://designsystem.digital.gov/components/tooltip/accessibility-tests/)
2. Scroll down to Screen reader tests
3. Note the "Screen reader announces tooltip content." item - it's listed as a failure

### Expected Behavior

The test status should be "passed."

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.
